RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplate2D.cc

```diff
@@ -1469,8 +1539,7 @@ void SiPixelTemplate2D::xysigma2(float qpixel, int index, float& xysig2)
             +adcotb_*(xypary1x0_[1][0]+xypary1x0_[1][1]*sigi+xypary1x0_[1][2]*sigi2+xypary1x0_[1][3]*sigi3+xypary1x0_[1][4]*sigi4 - err00);
          }
 			xysig2 =qscale*err2;

```

Comment:
We'll let xcode do that.

Reply:
Actually, I'll fix this by hand.  I don't trust xcode to do it, since it can introduce invisible changes that could make everything different.  But this will be done in the next commit.